configuration-service/handlers/project.go

```diff
@@ -177,6 +178,9 @@ func DeleteProjectProjectNameHandlerFunc(params project.DeleteProjectProjectName
 		}
 	}
 
+	mv := common.GetProjectsMaterializedView()
+	mv.DeleteProject(params.ProjectName)
```

Comment:
How can errors be handled? 
If the error is not handled, the materialized view and the state in the Git repo can potentially diverge.